ToolsQA/Pages/WikiPage.cs

```diff
@@ -0,0 +1,50 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using OpenQA.Selenium;
+using SeleniumExtras.PageObjects;
+using ToolsQA.Selenium_Basics;
+
+namespace ToolsQA.Pages
+{
+    class WikiPage
+    {
+        [FindsBy(How = How.CssSelector, Using = "[href*='/w/index.php?title=Specjalna:Zaloguj&returnto=Wikipedia%3AStrona+g%C5%82%C3%B3wna']")]
+        private IWebElement _loginButton;
+
+        [FindsBy(How = How.Id, Using = "searchInput")]
+        private IWebElement _searchTextBox;
+
+        [FindsBy(How = How.Id, Using = "searchButton")]
+        private IWebElement _searchButton;
+
+        [FindsBy(How = How.CssSelector, Using = "[href*='https://en.wikipedia.org/wiki/']")]
+        private IWebElement _languageButton;
```

Comment:
The name is misleading, it's not a button nor language. It's a hyperlink to a specific page.